cmd/web.go

```diff
@@ -229,6 +236,10 @@ func runWeb(ctx *cli.Context) error {
 		m.Post("/email/delete", user.DeleteEmail)
 		m.Get("/password", user.SettingsPassword)
 		m.Post("/password", bindIgnErr(auth.ChangePasswordForm{}), user.SettingsPasswordPost)
+		m.Get("/openid", user.SettingsOpenID)
+		m.Post("/openid", bindIgnErr(auth.AddOpenIDForm{}), user.SettingsOpenIDPost)
+		m.Post("/openid/delete", user.DeleteOpenID)
```

Comment:
It's another group (/setting). But used more Combo/Groups here too, again with same commit d3bebcf